pygmt/src/plot3d.py

```diff
@@ -78,11 +78,10 @@ def plot3d(
     x/y/z : float or 1d arrays
         The x, y, and z coordinates, or arrays of x, y and z coordinates of
         the data points
-    data : str or {table-like}
-        Pass in either a file name to an ASCII data table, a 2D
-        {table-classes}.
-        Use parameter ``columns`` to choose which columns are x, y, z, color,
-        and size, respectively.
+    data : str or 2d array
```

Comment:
Why change this back from {table-like} to only '2d array'?

Reply:
Not sure when or why I made this change. I'll change it back.

Reply:
Updated in b562d8b